src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/PreReleaseUserServiceTests.cs

```diff
@@ -918,6 +1158,127 @@ await using (var userAndRolesDbContext = DbUtils.InMemoryUserAndRolesDbContext(c
+                        Email = "test@test.com",
+                        EmailSent = false,
+                    },
+
+                    new User
+                    {
+                        Email = "test2@test.com"
+                    },
+                    new UserReleaseInvite
+                    {
+                        Release = release,
+                        Role = ReleaseRole.PrereleaseViewer,
+                        Email = "test2@test.com",
+                        EmailSent = true,
+                    }
+                );
+
+            await context.SaveChangesAsync();
+            }
+
+            var identityRole = new IdentityRole
+            {
+                Name = "Prerelease User"
+            };
+
+            await using (var userAndRolesDbContext = DbUtils.InMemoryUserAndRolesDbContext(contextId))
+            {
+                userAndRolesDbContext.Add(identityRole);
+                await userAndRolesDbContext.SaveChangesAsync();
+            }
+
+           var preReleaseService = new Mock<IPreReleaseService>();
+
+           preReleaseService
+               .Setup(s => s.GetPreReleaseWindow(It.IsAny<Release>()))
+               .Returns(
+                   new PreReleaseWindow
+                   {
+                       Start = DateTime.Parse("2020-09-08T08:30:00.00Z", styles: DateTimeStyles.AdjustToUniversal),
+                       End = DateTime.Parse("2020-09-08T22:59:59.00Z", styles: DateTimeStyles.AdjustToUniversal),
+                   }
+               );
+
+           var emailService = new Mock<IEmailService>();
+
+            await using (var context = DbUtils.InMemoryApplicationDbContext(contextId))
+            await using (var userAndRolesDbContext = DbUtils.InMemoryUserAndRolesDbContext(contextId))
+            {
+
+                var service = SetupPreReleaseUserService(
+                    context,
+                    usersAndRolesDbContext: userAndRolesDbContext,
+                    preReleaseService: preReleaseService.Object,
+                    emailService: emailService.Object
+                );
+
+                var result = await service.SendPreReleaseUserInviteEmails(release);
+
+                var sendMailResultDict = new Dictionary<string, dynamic>
+                {
+                    {"newUser", "no"},
+                    {"release name", "Academic Year 2020/21"},
+                    {"publication name", "Test publication"},
+                    {
+                        "prerelease link",
+                        $"http://localhost/publication/{release.PublicationId}/release/{release.Id}/prerelease/content"
+                    },
+                    {"prerelease day", "Tuesday 08 September 2020"},
+                    {"prerelease time", "09:30"},
+                    {"publish day", "Wednesday 09 September 2020"},
+                    {"publish time", "09:30"},
+                };
+
+                emailService.Verify(
```

Comment:
Seeing as how you're doing a manual Verify here over doing a Setup earlier, might be worth putting a `Once` or `Times(1)` on this to ensure it only sends an email once.